src/main/webapp/view.js

```diff
@@ -15,22 +15,54 @@
 const PARAM_ID = 'id';
 const PARAM_VIDEO_ID = 'video-id';
 
+let timer;
+/** Timestamp that video, disccussion, and transcript were synced to. */
+let lastSyncedTime;
+
 /* exported LECTURE_ID */
 window.LECTURE_ID = getLectureId();
 window.VIDEO_ID = getVideoId();
 
 initialize();
 
 /**
- * Initializes the video player and discussion section
+ * Initializes the timer, video player, and discussion section
  * for the lecture view page.
  */
 async function initialize() {
   window.loadVideoApi();
-  window.intializeDiscussion();
+  window.loadDiscussion();
+  // TODO: Adjust time interval.
+  timer = window.setInterval(getCurrentTime, 1000);
   // TODO: Initialize the trancript section.
 }
 
+/**
+ * Retrives the current time of the video and relays time to
+ * {@code sync}.
+ */
+function getCurrentTime() {
```

Comment:
can this be updated to `getCurrentVideoTime`? I think it can get a bit confusing later since the comments and transcripts also have timestamps. Also, can you specify the time units for this in the method header?